src/Sylius/Bundle/ProductBundle/spec/EventListener/SelectProductAttributeChoiceRemoveListenerSpec.php

```diff
@@ -67,6 +68,8 @@ function it_removes_select_product_attribute_choices(
             ],
         ]);
 
+        $connection->getDatabasePlatform()->willReturn(new MySqlPlatform());
```

Comment:
Could you please add more context why this change is necessary.
For example there is similar test at line 125 of this file -- does it require platform setup as well?